### PR TITLE
ICMSLST-3015 Add/change bought from details when app is out for update

### PR DESCRIPTION
**Pull request by mikeylockwood**

### CODE REVIEW

File: web/domains/case/_import/fa/views.py [hunk 1]
```diff
@@ -124,9 +125,19 @@ def create_import_contact(
...
+                    application, [ImpExpStatus.IN_PROGRESS, ImpExpStatus.PROCESSING]
+                )
+            except errors.ProcessStatusError as e:
+                # Check if application is out for update with ownership released
+                if not (
+                    application.status == ImpExpStatus.SUBMITTED
+                    and not application.case_owner
+                    and application.current_update_requests().exists()
+                ):
+                    raise e
```
Comment (MattHolmes123) [inline web/domains/case/_import/fa/views.py:139]: Can you do this check in a single function in this file please?
I will be refactoring it soon enough anyway but no need for the duplication

Comment (MattHolmes123) [reply web/domains/case/_import/fa/views.py:139]: Actually Michael... can it just use this: `case_progress.application_in_progress(application)`

Comment (MattHolmes123) [reply web/domains/case/_import/fa/views.py:139]: image

**Review by MattHolmes123 — APPROVED** [review]

File: web/domains/case/_import/fa/views.py [hunk 2]
```diff
@@ -234,11 +231,24 @@ def delete_import_contact(
...
-        )
-
+        try:
+            case_progress.check_expected_status(
+                application,
+                [
+                    ImpExpStatus.IN_PROGRESS,
+                    ImpExpStatus.PROCESSING,
+                    ImpExpStatus.VARIATION_REQUESTED,
+                    ImpExpStatus.COMPLETED,
```
Comment (mikeylockwood) [inline web/domains/case/_import/fa/views.py:241]: This also has a requirement fro completed as the view is used in the supplementary report

**Pull request merged**